python/tests/util.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import absolute_import, print_function, unicode_literals
+from builtins import dict, str
+
+import sys
+import unittest
+from os import path
+
+import kappy
+from kappy.kappa_common import KASIM_DIR
+
+MODELS_DIR = path.join(KASIM_DIR, "models")
```

Comment:
Models are not installed either. So KASIM_DIR is not the good path is kappy is installed